requirements/base.txt

```diff
@@ -24,6 +24,7 @@ chardet==3.0.4            # via aiohttp
 click==7.1.2              # via apache-superset, flask, flask-appbuilder
 colorama==0.4.4           # via apache-superset, flask-appbuilder
 contextlib2==0.6.0.post1  # via apache-superset
+convertdate==2.2.0        # via holidays
```

Comment:
current python-holidays depend on convertdate == 2.2.0, convertate depend on  'pytz>=2014.10, < 2020'.  Conflict with the current pytz version(pytz==2020.4), but python-holidays community is trying to fix the problem. 

https://github.com/dr-prodigy/python-holidays/issues/406